docs/src/main/asciidoc/security-webauthn.adoc

```diff
@@ -619,16 +619,17 @@ creation and logins yourself via your own logic (see <<Handling login and regist
 
 The Quarkus WebAuthn extension comes out of the box with these REST endpoints pre-defined:
 
-=== Obtain a registration challenge
```

Comment:
to me it feels the primary part here is how to get the challenge - and that it is also options is secondary so I would keep the challenge part in title.